.circleci/config.yml

```diff
@@ -1016,6 +1033,35 @@ jobs:
 
                       # Docker logout
                       docker logout
+                      
+            - when:
+                  condition:
+                      equal: [false, << parameters.dry_run >>]
+                  steps:
+                      - run:
+                          name: graviteeio/apim-management-api
+                          command: |
+                            export IMAGE1="graviteeio/apim-management-api"
+                            echo "export IMAGE1=$IMAGE1" >> $BASH_ENV
+                      - docker-scan                   
+                      - run:
+                          name: graviteeio/apim-gateway
+                          command: |
+                            export IMAGE1="graviteeio/apim-gateway"
+                            echo "export IMAGE1=$IMAGE1" >> $BASH_ENV
+                      - docker-scan 
+                      - run:
+                          name: graviteeio/apim-management-ui
+                          command: |
+                            export IMAGE1="graviteeio/apim-management-ui"
+                            echo "export IMAGE1=$IMAGE1" >> $BASH_ENV
+                      - docker-scan 
+                      - run:
+                          name: graviteeio/apim-portal-ui
+                          command: |
+                            export IMAGE1="graviteeio/apim-portal-ui"
+                            echo "export IMAGE1=$IMAGE1" >> $BASH_ENV
+                      - docker-scan                       
```

Comment:
If we use parameters in the command we don't need all these environment variables:

```suggestion

            - when:
                  condition:
                      equal: [false, << parameters.dry_run >>]
                  steps:
                      - docker-scan:
                            docker-image-name: graviteeio/apim-management-api
                            version: << parameters.graviteeio_version >><<# parameters.enterprise_edition >>-ee<</ parameters.enterprise_edition >>
                      - docker-scan:
                            docker-image-name: graviteeio/apim-gateway
                            version: << parameters.graviteeio_version >><<# parameters.enterprise_edition >>-ee<</ parameters.enterprise_edition >>
                      - docker-scan:
                            docker-image-name: graviteeio/apim-management-ui
                            version: << parameters.graviteeio_version >><<# parameters.enterprise_edition >>-ee<</ parameters.enterprise_edition >>
                      - docker-scan:
                            docker-image-name: graviteeio/apim-portal-ui
                            version: << parameters.graviteeio_version >><<# parameters.enterprise_edition >>-ee<</ parameters.enterprise_edition >>
                     
```